api/http/common.go

```diff
@@ -20,62 +20,63 @@ import (
-	LastNameKey      = "last_name"
-	TotalKey         = "total"
-	SubjectKey       = "subject"
-	ObjectKey        = "object"
-	LevelKey         = "level"
-	StartLevelKey    = "start_level"
-	EndLevelKey      = "end_level"
-	TreeKey          = "tree"
-	DirKey           = "dir"
-	ListPerms        = "list_perms"
-	VisibilityKey    = "visibility"
-	EmailKey         = "email"
-	SharedByKey      = "shared_by"
-	TokenKey         = "token"
-	UserKey          = "user"
-	DomainKey        = "domain"
-	ChannelKey       = "channel"
-	ConnTypeKey      = "connection_type"
-	DefPermission    = "read_permission"
-	DefTotal         = uint64(100)
-	DefOffset        = 0
-	DefOrder         = "updated_at"
-	DefDir           = "asc"
-	DefLimit         = 10
-	DefLevel         = 0
-	DefStartLevel    = 1
-	DefEndLevel      = 0
-	DefStatus        = "enabled"
-	DefClientStatus  = clients.Enabled
-	DefUserStatus    = users.Enabled
-	DefGroupStatus   = groups.Enabled
-	DefListPerms     = false
-	SharedVisibility = "shared"
-	MyVisibility     = "mine"
-	AllVisibility    = "all"
+	MemberKindKey       = "member_kind"
+	PermissionKey       = "permission"
+	RelationKey         = "relation"
+	StatusKey           = "status"
+	OffsetKey           = "offset"
+	OrderKey            = "order"
+	LimitKey            = "limit"
+	MetadataKey         = "metadata"
+	ParentKey           = "parent_id"
+	OwnerKey            = "owner_id"
+	ClientKey           = "client"
+	UsernameKey         = "username"
+	NameKey             = "name"
+	GroupKey            = "group"
+	ActionKey           = "action"
+	ActionsKey          = "actions"
+	RoleIDKey           = "role_id"
+	RoleNameKey         = "role_name"
+	AccessProviderIDKey = "access_provider_id"
+	AccessTypeKey       = "access_type"
+	TagKey              = "tag"
+	FirstNameKey        = "first_name"
+	LastNameKey         = "last_name"
+	TotalKey            = "total"
+	SubjectKey          = "subject"
+	ObjectKey           = "object"
+	LevelKey            = "level"
+	StartLevelKey       = "start_level"
+	EndLevelKey         = "end_level"
+	TreeKey             = "tree"
+	DirKey              = "dir"
+	ListPerms           = "list_perms"
+	VisibilityKey       = "visibility"
+	EmailKey            = "email"
+	SharedByKey         = "shared_by"
+	TokenKey            = "token"
+	UserKey             = "user"
+	DomainKey           = "domain"
+	ChannelKey          = "channel"
+	ConnTypeKey         = "connection_type"
+	DefPermission       = "read_permission"
+	DefTotal            = uint64(100)
+	DefOffset           = 0
+	DefOrder            = "updated_at"
+	DefDir              = "asc"
+	DefLimit            = 10
+	DefLevel            = 0
+	DefStartLevel       = 1
+	DefEndLevel         = 0
+	DefStatus           = "enabled"
+	DefClientStatus     = clients.Enabled
+	DefUserStatus       = users.Enabled
+	DefGroupStatus      = groups.Enabled
+	DefListPerms        = false
+	SharedVisibility    = "shared"
+	MyVisibility        = "mine"
+	AllVisibility       = "all"
```

Comment:
Can we logically group these keys (per API?) so we split this large const block?